bigtable/client/row_reader.cc

```diff
@@ -81,6 +81,46 @@ RowReader::RowReader(
       raise_on_error_(raise_on_error),
       error_retrieved_(raise_on_error) {}
 
+RowReader::RowReader(
+    std::shared_ptr<DataClient> client, std::string app_profile_id,
+    std::string table_name, RowSet row_set, std::int64_t rows_limit,
+    Filter filter, std::unique_ptr<RPCRetryPolicy> retry_policy,
+    std::unique_ptr<RPCBackoffPolicy> backoff_policy,
+    MetadataUpdatePolicy metadata_update_policy,
+    std::unique_ptr<internal::ReadRowsParserFactory> parser_factory)
+    : RowReader(std::move(client), std::move(app_profile_id),
+                std::move(table_name), std::move(row_set), rows_limit,
+                std::move(filter), std::move(retry_policy),
+                std::move(backoff_policy), std::move(metadata_update_policy),
+                std::move(parser_factory), true) {}
+
+RowReader::RowReader(
+    std::shared_ptr<DataClient> client, std::string app_profile_id,
+    std::string table_name, RowSet row_set, std::int64_t rows_limit,
+    Filter filter, std::unique_ptr<RPCRetryPolicy> retry_policy,
+    std::unique_ptr<RPCBackoffPolicy> backoff_policy,
+    MetadataUpdatePolicy metadata_update_policy,
+    std::unique_ptr<internal::ReadRowsParserFactory> parser_factory,
+    bool raise_on_error)
+    : client_(std::move(client)),
```

Comment:
No action from you, but this class has a really big number of arguments...  Seems like we would want to group a few, the policies, the arguments for iteration, the client and the profile id.